protocol/cosmic-features.json

```diff
@@ -0,0 +1,927 @@
+      "0084-VDPR-014",
+      "0084-VDPR-015",
+      "0084-VDPR-016",
+      "0084-VDPR-017",
+      "0029-FEES-023",
+      "0029-FEES-024",
+      "0029-FEES-025",
+      "0029-FEES-026",
+      "0029-FEES-027",
+      "0029-FEES-028",
+      "0029-FEES-029",
+      "0029-FEES-030",
+      "0029-FEES-031",
+      "0029-FEES-032",
+      "0029-FEES-033",
+      "0062-SPAM-026",
+      "0062-SPAM-027",
+      "0062-SPAM-028",
+      "0062-SPAM-029",
+      "0062-SPAM-030",
+      "0062-SPAM-031",
+      "0062-SPAM-032",
+      "0062-SPAM-033",
+      "0062-SPAM-034",
+      "0062-SPAM-035",
+      "0062-SPAM-036",
+      "0062-SPAM-037"
+    ]
+  },
+"Rewards": {
+    "milestone": "deployment-2",
+    "acs": [
+      "0085-RVST-001",
+      "0085-RVST-002",
+      "0085-RVST-003",
+      "0085-RVST-004",
+      "0085-RVST-005",
+      "0085-RVST-006",
+      "0085-RVST-007",
+      "0085-RVST-008",
+      "0085-RVST-009",
+      "0085-RVST-010",
+      "0085-RVST-011",
+      "0085-RVST-012",
+      "0085-RVST-013",
+      "0085-RVST-014",
+      "0086-ASPR-001",
+      "0086-ASPR-002",
+      "0086-ASPR-003",
+      "0086-ASPR-004",
+      "0086-ASPR-005",
+      "0086-ASPR-006",
+      "0086-ASPR-007",
+      "0086-ASPR-008",
+      "0086-ASPR-009",
+      "0086-ASPR-010",
+      "0056-REWA-076",
+      "0056-REWA-077",
+      "0056-REWA-078",
+      "0056-REWA-079",
+      "0056-REWA-080",
+      "0056-REWA-081",
+      "0056-REWA-082",
+      "0056-REWA-083",
+      "0056-REWA-084",
+      "0056-REWA-085",
+      "0056-REWA-086",
+      "0056-REWA-087",
+      "0056-REWA-088",
+      "0056-REWA-089",
+      "0056-REWA-090",
+      "0056-REWA-091",
+      "0056-REWA-092",
+      "0056-REWA-093",
+      "0056-REWA-094"
+    ]
+  },
+  "Team rewards": {
+    "milestone": "deployment-3",
+    "acs": [
+      "0056-REWA-095",
+      "0056-REWA-096",
+      "0056-REWA-097",
+      "0056-REWA-098",
+      "0056-REWA-099",
+      "0056-REWA-100",
+      "0056-REWA-101",
+      "0056-REWA-102",
+      "0056-REWA-103",
+      "0056-REWA-104",
+      "0056-REWA-105",
+      "0056-REWA-106",
+      "0056-REWA-107",
+      "0056-REWA-108"
+    ]
+  },
+  "Market governance": {
+    "milestone": "deployment-2",
+    "acs": [
+      "0028-GOVE-064",
+      "0028-GOVE-069",
+      "0028-GOVE-070",
+      "0028-GOVE-072",
+      "0028-GOVE-108",
+      "0028-GOVE-110",
+      "0028-GOVE-113",
+      "0028-GOVE-114",
+      "0028-GOVE-115",
+      "0028-GOVE-116",
+      "0028-GOVE-117",
+      "0028-GOVE-118",
+      "0028-GOVE-135",
+      "0028-GOVE-136",
+      "0028-GOVE-137",
+      "0028-GOVE-138",
+      "0028-GOVE-139",
+      "0028-GOVE-111",
+      "0028-GOVE-150",
+      "0028-GOVE-151",
+      "0028-GOVE-152"
+    ]
+  },
+    "Batch change proposals": {
+    "milestone": "deployment-3",
+    "acs": [
+      "0028-GOVE-145",
+      "0028-GOVE-146",
+      "0028-GOVE-147",
+      "0028-GOVE-148",
+      "0028-GOVE-149"
+    ]
+  },
+  "Spot": {
```

Comment:
Not sure this section should be here.

Reply:
I left this in so we had a record of all the spot ACs for when spot gets prioritised.